lib/autofix.js

```diff
@@ -106,9 +106,8 @@ function formatFileContent(options, file, filePath) {
   if (options.elmFormatPath) {
     const spawnedUsingPathFromArgs = spawn.sync(
       options.elmFormatPath,
-      ['--elm-version=0.19', '--stdin', '--output', `"${filePath}"`],
+      ['--elm-version=0.19', '--stdin', '--output', filePath],
       {
-        shell: true,
         input: file.source
```

Comment:
Separately, I believe `spawn.sync` has no `input` option. I don’t think they hurt, though, but they could probably be removed at some point (not this PR).